ckanext/switzerland/helpers/plugin_utils.py

```diff
@@ -1,24 +1,22 @@
 """
 helpers of the plugins.py
 """
-import isodate
 import json
-import re
 import logging
-from ckan import logic
+import re
+
 import ckan.plugins.toolkit as toolkit
```

Comment:
`import ckan.plugins.toolkit as tk` ?

Reply:
:joy: @sarusarah suggested the reverse change for a different file that already had `import ckan.plugins.toolkit as tk`.

Personally I prefer `tk`, because it's shorter, we usually import it this way (11 out of 13 times in this repo), and it's used very often, so we all know what it stands for. (I do usually prefer descriptive names over short ones, though.) I will change it to `tk` in these two files so it's consistent in our code, unless anyone really objects.